core/src/main/javascript/io/Url.js

```diff
@@ -341,6 +351,9 @@
                                     throw this._exception;
                                 }
 
+                                // Capture headers
+                                this._headers = result.response.headers;
```

Comment:
Need to change the callback response value of HttpOutputStream to include the headers property.

`{{result: function({=response: {=body: string, =content: Uint8Array, =headers: ?Array<string>}, =isTimeout: boolean, =isError: boolean}), error: function(Error)}}`